lava/magma/core/model/c/ports_python.c

```diff
@@ -0,0 +1,43 @@
+#define PYTHON
+#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
+#include <Python.h>
+#include "numpy/arrayobject.h"
+#include <assert.h>
+#include <stdint.h>
+#include "ports.h"
+
+size_t send(PyObject *port, void *data, size_t m){ // send via pointer
```

Comment:
might want to change this at some point to get 'm' from the port object. doing it this way for now to make one less thing that could go wrong.